hack/generator/pkg/codegen/pipeline_export_generated_code.go

```diff
@@ -87,33 +88,58 @@ func writeFiles(ctx context.Context, packages map[astmodel.PackageReference]*ast
 	globalProgress := newProgressMeter()
 	groupProgress := newProgressMeter()
 
+	var wg sync.WaitGroup
+
+	wgDone := make(chan bool)
```

Comment:
minor: Consider a comment about why parallel here - possible link to issue/issues in `dst` (if that's the primary cause of problems you noticed)